### PR TITLE
Add downloads and history functionality

### PR DESCRIPTION
**Pull request by srvkmr130**
add functionality for

downloads
history

### CODE REVIEW

**Comment by gauthamzz** [comment]
please wirte commit message as add functionality for downloads or something else ,not update background.js in the future

File: background.js [hunk 1]
```diff
@@ -211,6 +211,15 @@ $(document).ready(function () {
           if (data.result.metadata.intentName === "open") {
             chrome.tabs.create({ 'url': "http://www." + data.result.parameters.website });
           } else
+		if (data.result.metadata.intentName === "history") {
+chrome.tabs.create({'url': 'chrome://history'});
```
Comment (gauthamzz) [inline background.js:215]: do you mind indenting the code properly

**Review by gauthamzz — APPROVED** [review]
This looks good to me, can you intend properly . I will merge the pr

File: background.js (hunk 1 above)
Comment (srvkmr130) [reply background.js:215]: okay I will try

**Pull request closed without merging**